private/buf/buflsp/symbol.go

```diff
@@ -253,6 +254,11 @@ func (s *symbol) FormatDocs() string {
 		builtin, _ := s.kind.(*builtin)
 		comments, ok := builtinDocs[builtin.predeclared.String()]
 		if ok {
+			comments = append(
+				comments,
+				"",
+				"This symbol is a Protobuf builtin. [Learn more on protobuf.com.](https://protobuf.com/docs/language-spec#field-types)",
```

Comment:
nit: should we `s/This symbol/<symbol-name>`, for consistency with the below link?